### PR TITLE
Vox can wear normal hardsuits, removes suits from roundstart Vox except Trader.

### PR DESCRIPTION
**Pull request by ghost**
### Still being worked on

Adds a "Tank Harness" (invisible on mob) suit slot item that Vox spawn with which can hold tanks, removes all roundstart Vox suits except for Trader. Job hardsuits (Med, Sec, Eng, Atmos, CE, Mining) and the Paramedic EVA Suit can now be worn by Vox, no need for a special Vox-only one.

Do not merge yet as I need help with SQL/Preferences stuff. @PJB3005 

:cl:
 * rscadd: Added Tank Harness to all Vox except Trader on spawn for holding their tank
 * rscdel: Removed roundstart suits from all Vox except Trader
 * tweak: Vox can now wear regular hardsuits (sec, engi, atmos, CE, mining, medical)

### CODE REVIEW

File: code/game/objects/items/weapons/tanks/tank_types.dm [hunk 1]
```diff
@@ -129,15 +142,14 @@
...
-	name = "nitrogen tank"
-	desc = "A tank of nitrogen."
-	icon_state = "oxygen_fr"
-	distribute_pressure = ONE_ATMOSPHERE*O2STANDARD
+/obj/item/weapon/tank/emergency_nitrogen/vox
+	name = "vox specialized nitrogen tank"
+	desc = "A high-tech nitrogen tank designed specifically for Vox."
+	flags = FPRINT
+	icon_state = "emergency_vox"
+	force = 4.0
```
Comment (PJB3005) [inline code/game/objects/items/weapons/tanks/tank_types.dm]: Remove redundant decimals.

**Review by PJB3005 — CHANGES_REQUESTED** [review]
You need to add a migration datum too.

File: code/game/objects/items/weapons/tanks/tank_types.dm (hunk 1 above)
Comment (ghost) [reply code/game/objects/items/weapons/tanks/tank_types.dm]: That's a non standard formatting change :^)

**Comment by Blithering** [comment]
Could you add the small plasma tanks while you're poking at this? Plasmamen still spawn with small oxygen tanks in their emergency box.

**Comment by ghost** [comment]
So vox can get free effectively unlimited air in a slot that fits in their pocket, now?

**Comment by Allspice** [comment]
Why would birds get any "high-tech" things half of them talk like retards in their racial language

**Comment by SonixApache** [comment]
Large tanks that fit in your pocket sound pretty crazy.

**Comment by yclatious** [comment]
Why would you make the biggest flaw of the vox, their need of a suit to hold the air tank, completely null by just making a big one into a pocket sized one.

Vox are underequiped space gypsie jews, and a race with flaws, they already have no slowdown magboots, access to a lot of sec gear for free, engie stuff, their own botany with exclusive shit, and now you want to null one of their weakest point too, the availability of nitrogen and the portability of big tanks, on what reason?

Cause if you think vox need a buff, they fucking don't, and if this is some stupid as fuck buff just because you play them, Ill tell you to fuck off. Other races are gimped for a reason, vox already got tons of free shit, powerful stuff too, this is just a plain buff and powercreep by racefags, you have tons of emergency nitrogen tanks in the station, or if you really want to be safe, maybe trade a extended oxy tank with the crew so you can store your nitrogen there, instead of codding in a buff to bird shitters with no basis for it.

Vox a pox, race war now.

**Comment by Kurfursten** [comment]
@uraniummeltdown I think you are mistaken, this is the /vg/ repo, you may have been intending to PR to your usual repo, which is TG.

I'm a bit bewildered as to why a coder switches servers and only wants to contribute balance changes.

**Comment by ghost** [comment]
The reason I'm doing this is because I intend to remove roundstart Vox suits for all jobs except Trader and make it so Vox can wear the human hardsuits on-station as well (just like with the blood-red hardsuit) If humans don't get roundstart suits Vox shouldn't either.

**Comment by Exxion** [comment]
Duny did that once
Pomf reverted it

**Comment by Exxion** [comment]
Also, Fun Fact:
Vox don't actually need space suits as long as they have internals and something to keep them warm
Old holdover from Bay that apparently nobody bothered changing

**Comment by Ephx1** [comment]
Remove the only upside of being a bird
Great
Fantastic

Next you could remove plasmaman suits, and skeletons being able to move at all.

**Comment by Exxion** [comment]
Vox pros:
- Get a space suit
- Can conceivably get no-slowdown magboots, though the number available is very limited
- The nitrogen reagent heals suffocation damage and toxin damage for them at an average rate but for a very long time
- Gravy heals toxin damage for them, but isn't very good at it
- Don't take low-pressure damage (but their lungs still pop if they don't have internals)
  - This one should be changed
- Slightly increased kick damage while not wearing shoes
- Can butcher things without a tool if they aren't wearing a mask, albeit much more slowly than with a tool

As you can see, they have a bunch of pretty minor advantages and a couple moderate-to-major ones.

Vox cons:
- Oxygen is highly toxic to them. More toxic than plasma (by a wide margin, in my experience).
- Inaprov is toxic to them and doesn't work on them (other than its lesser-known effects on internal bleeding)
- They have to carry around a nitrogen tank or figure out some other scheme to not die on the station.
  - They can carry it on their backs (preventing them from using a backpack) or in their suit storage (preventing them from wearing armor, and also slowing them down substantially)
  - This can be avoided with advanced lungs or a bunch of nitrogen plus a shit ton of antitox (neither heals quickly enough on its own), but that requires you actually get one of those
  - If someone removes their tank and immobilizes them, they'll die within two minutes.
- No Asimov protection
- Can't wear most hardsuits

And as you can also see, they also have a bunch of weaknesses, and a couple major ones.

Balancing the races isn't about making them as identical as possible. That's boring. It's about balancing their strengths with weaknesses and vice versa.
Vox are currently pretty well-balanced. They are not, in general, any more or less likely to survive all the way through a round than humans are

**Comment by ghost** [comment]
Alright @Sprok0 gave me the idea of a tank harness that goes in the Suit Storage slot, that sounds better than adding pocket tanks, I'll work on it

**Comment by Intigracy** [comment]
I didn't know about the immune to low pressure and agree it should be removed at some point, just not in this PR.

If you're gonna remove the roundstart suits from vox, you need to add a way to acquire some.

**Comment by ghost** [comment]
@Intigracy Swiping ID on the Assistant suits will turn it into the job suit, for example the Chef can go order a Vox suit, swipe his ID on it and get the Chef one with the hat

Does not apply to jobs with hardsuits like Engineering Hardsuit, Sec Hardsuit, etc.

**Comment by yclatious** [comment]
Why not just add some in cargo, make  service, engineering, medical and sec vox suit crates instead?

Plus, how would Id suit swiping work, could they just steal a captain or sec Id, swipe it and get a free sec gear? Is it based on access or title, because if they get extra access, how would the suit be chosen, if they for example have engineering and science acess?

**Comment by SonixApache** [comment]
Add a xenovend vending machine that dispenses special items for xenos. Standard ID check only works for aliens unless hacked.
Speaking of; add species as a field to ID cards.

**Comment by yclatious** [comment]
Or we could just not give away extremely powerful items that negate the downsides of races for the sake of removing suits that don't actually impact them much.

**Comment by kilozombie** [comment]
I play trader half the time I join and I agree with people in this thread. You can't just nullify what makes a Vox disadvantaged and frustrating to play, at least not with this sort of item.

**Comment by Intigracy** [comment]
My plan at some point in the future is to redesign space / hardsuits to have two mod vars, one for the look and stats (like engineering being yellow and having some rad resistance) and one for which species it fits, with these being removable (no species mod = fits everyone that fits in human suits) / replacable / orderable along with generic suits/hardsuits (maybe having to make hardsuits yourself?)

**Comment by ghost** [comment]
"my plan at some point in the future" means never

**Comment by Kurfursten** [comment]
I can't believe anyone would want to give up our neat Vox suit sprites. I don't even play Vox but I think some of our Vox suits (especially Chef) are really nice.

**Comment by Ephx1** [comment]
It's a shame they're so nice, gypsy birds need the bare minimum, but the sprites are so good not to use.

**Comment by Exxion** [comment]
@Kurfursten 

> Swiping ID on the Assistant suits will turn it into the job suit, for example the Chef can go order a Vox suit, swipe his ID on it and get the Chef one with the hat
> 
> Does not apply to jobs with hardsuits like Engineering Hardsuit, Sec Hardsuit, etc.

I'm still firmly against this PR but that thing in particular isn't a problem

**Comment by Intigracy** [comment]
This is now an even worse change than it was.

**Comment by Ephx1** [comment]
@Exxion 
Well I'll be, most of those upsides are pretty pointless but some of it is cool.
Most people know about the suit though.

**Comment by Intigracy** [comment]
Specifically the harness is fine as an idea, but vox being able wear every space humans can is pants on head retarded.

**Comment by yclatious** [comment]
Vox are only able to get the blood red suit for the sake of actually playing Nukeops, I still don't think its good to removing our well sprited suits, to make them wear any engineering suits or make shit compatible to them, adding a extra harness just so you don't use the suits and essentially give away a free large invisible canister holster so you can parade around in fucking normal clothing.

Vox are played in a specific manner, and their dependance on nitrogen is crucial to their role. You want to essentially make them be able to just be walking normal people with no alien characteristics or apearences, like some fucking furfag snowflake who plays as a vox with a executive suit, just uses a invisible harness for his large nitrogen tank and walks about with just a mask, trying to fit in? Do we really need furfag snowflake aceptance or making it in any way easier to be one, when one of the major things about Vox is how they are not at all adapted to the station, how they are different from the crew they are selling to, how they are, even by their fellow coworkers, rearded as alien creatures, because they effectively are alien as fuck when compared to humans?

Maybe its just me, but this just reeks of useless aditions that can cause stupid as fuck shit. Vox are to have inherent flaws in their design, they are underequiped aliens in a station filled with people that do not like the, filled with extremely toxic air to them, and a lot of aggressiveness at the smallest mistake, you play vox to be a alien in a human station, not to integrate yourself and blend in, you are a bipedal chicken raptor who cannot breathe oxygen, cannot actually speak Common decently and is generally a dumb kleptomaniac, making you ditch the suit and trying to hide your naturally alien apearence makes no sense, your suit is needed, it defines vox, it makes it clear that is a alien race, a widely ostracized and unfairly treated race that ocasionaly shines through and proves to be a good bunch, be it a nice bartender, a good card dealer, anything, but that shit is done naturaly, and it still shows how alien they are, and doesn't try to hide it with some bollocks about a special satchel that hides a rather large nitrogen tank or the sake of hiding it and fitting in. Maybe Im spouting bollocks, but I still do not see a single way this will actually improve the currently playing Vox in any reasonable way other than making them lose the natural Alien-ness they have for the sake of snowflake shit.

**Comment by jakmak3** [comment]
Funfact about birds they are 100% resistant to the effects of a hard vacuum (space) but are not hypomemia proof (the cold meme)

**Comment by Praecordia** [comment]
seems sorta pointless desu

**Comment by Durgen** [comment]
Why add a special snowflaked slot when you could give vox extended capacity, pocket sized nitrogen tanks.

**Comment by kilozombie** [comment]
I'd not mind that. At least it's not balls-to-the-wall insane.

**Comment by Exxion** [comment]
@jakmak3 They can still get popped lungs.
Also I already said that. Twice.

**Comment by Arthurdentist** [comment]
Whilst we are at it I think we should also add a special 'shoe harness' so that catbeasts and lizards can wear shoes in their pockets to protect them against glass shards.

**Comment by Shadowmech88** [comment]
Glove harness for golems when?

**Comment by Kurfursten** [comment]
@Shadowmech88 Given their location, I think a borer harness for golems makes more sense. This would be a big hunk of flesh that the golem straps to himself so that a borer can live inside.

**Comment by SonixApache** [comment]
golems as actual races when

**Comment by Intigracy** [comment]
Glove harness after riot shield harness for sec.

**Comment by yclatious** [comment]
Great addition, everyone is loving these tg additions for snowflake furfags, what else, minibombs and pizza bombs :^) ?

**Comment by Shadowmech88** [comment]
@SonixApache 
Golems were made into a proper race earlier this year.

**Comment by Exxion** [comment]
Wow, I think this is the only PR ever to receive this many thumbs down without a single thumb up

**Comment by SonixApache** [comment]
I like the idea of removing roundstart vox suits and adding harnesses, though.

**Comment by kilozombie** [comment]
That seems to be all this PR is now.
Vox now start with a suit slot harness, that lets them have a suit storage tank, but prevents armor. They also don't start with a hardsuit but can put others on or order some Vox-specific ones from Cargo.
I'm not 100% on this change but it's not like it was originally, with a snowflake slot.

**Comment by Intigracy** [comment]
`>That seems to be all this PR is now`
`>Vox can wear normal hardsuits`

**Comment by kilozombie** [comment]
I don't think that's good, but the alternative is either having pods in EVA for vox suits, which just feels like a lot of clutter for no reason, or forcing all Vox to rely on Cargo for their hardsuits, which seems pretty irritating.

**Comment by Arthurdentist** [comment]
its as much about the journey as the destination.

**Comment by kilozombie** [comment]
In terms of the PR? Yeah.
I don't like a lot of this guy's suggested changes, but people have been asking for /tg/ ports for a while now. The problem is that a lot of portable ideas aren't ones fit for /vg/.

**Comment by SonixApache** [comment]
vox
vending
machines

**Comment by ghost** [comment]
not now, maybe some other time

**Pull request closed without merging**